app/Main.hs

```diff
@@ -204,7 +235,7 @@ optsParserInfo =
               $gitBranch,
               $gitHash
             ],
-          "using ghc-lib-parser " ++ VERSION_ghc_lib_parser
+          "using ghc-lib-parser " ++ "9.2.1.20211101"
```

Comment:
This should be reverted.

Reply:
I don't know what this line is, and I don't remember making this change... Maybe I missclicked in my IDE? :thinking:

Reply:
Fixed!